core/http/src/main/java/org/trellisldp/http/AgentAuthorizationFilter.java

```diff
@@ -62,6 +62,14 @@
     private final AgentService agentService;
     private final Set<String> adminUsers;
 
+    /**
+     * No-op constructor for CDI.
+     */
+    AgentAuthorizationFilter() {
+        this.agentService = null;
```

Comment:
Ditto comment above.

Reply:
This constructor is actually fine -- it's package-protected in scope, so no one will be able to invoke it outside of the `org.trellisldp.http` package.

Reply:
I disagree-- I think we should not a have a ctor that doesn't actually work in the codebase, period. It is confusion waiting to happen down the line.

Reply:
What about this [commit](https://github.com/trellis-ldp/trellis/commit/f8612bc63a1574b4e3fe1638b33ab3bd95a5b38f)? We are adding these sorts of constructors elsewhere. Why not here, too?

Reply:
OUCH! I was wrong then-- I got into too much of a hurry. To me _this_ exactly where the no-op pattern _does_ makes sense.